index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Apple Clone</title>
+    <link rel="stylesheet" href="assets/css/normalize.css" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@200;300;600&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/d86848cfe0.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <main>
+        <article class="bg-img">
+            <nav class="nav">
+                <ul class="flex">
+                    <li class="left-radius hover-effect">
+                        <a href="#"><i class="fab fa-apple"></i></a>
+                    </li>
+                    <li class="hover-effect"><a href="#">Store</a></li>
+                    <li class="hover-effect"><a href="#">Mac</a></li>
+                    <li class="hover-effect"><a href="#">iPod</a></li>
+                    <li class="hover-effect"><a href="#">iPhone</a></li>
+                    <li class="hover-effect"><a href="#">iPad</a></li>
+                    <li class="hover-effect"><a href="#">iTunes</a></li>
+                    <li class="hover-effect"><a href="#">Support</a></li>
+                    <li>
+                        <div class="flex search">
+                            <i class="fa fa-search" aria-hidden="true"></i>
+                            <input type="search" />
+                        </div>
+                    </li>
+                </ul>
+            </nav>
+            <section class="grid main">
+                <section class="headings">
+                    <h1>What will your verse be?</h1>
+                    <h2>
+                        Everything has something to share.See how people are using iPad in
+                        their unique ways to add to the world's story.
+                    </h2>
+                    <h3>
+                        Watch the "YourVerse" film and explore the stories
+                        <i class="fa fa-play-circle-o" aria-hidden="true"></i>
+                    </h3>
+                </section>
+            </section>
+        </article>
+        <article class="promotions flex">
+            <img src="assets/image/footer_img1.jpg" alt="footer image one">
+            <img src="assets/image/footer_img2.jpg" alt="footer image two">
+            <img src="assets/image/footer_img3.jpg" alt="footer image three">
+            <img src="assets/image/footer_img4.jpg" alt="footer image four">
+        </article>
```

Comment:
- [ ] Please fix the horizontal scroll that is being caused because your promotions flex is too big.

![image](https://user-images.githubusercontent.com/53256139/88675297-68d78e00-d0b0-11ea-957c-b84a5725976b.png)